### PR TITLE
To include timestamp field in the micrortps bridge messages.

### PR DESCRIPTION
**Pull request by vicentemonge**
Under petition of community and particularly of @avinash-palleti (Intel). @bkueng

### CODE REVIEW

File: msg/templates/uorb_microcdr/msg.cpp.template [hunk 1]
```diff
@@ -151,6 +147,7 @@ void serialize_@(topic_name)(const struct @(uorb_struct) *input, char *output, u
 
     resetStaticMicroCDRForSerialize(microCDRWriter);
 
+    serializeUnsignedLong(input->timestamp, microCDRWriter);
```
Comment (bkueng) [inline msg/templates/uorb_microcdr/msg.cpp.template]: Should this not be `serializeUnsignedLongLong`?

Comment (vicentemonge) [reply msg/templates/uorb_microcdr/msg.cpp.template]: No, is the correct one. Please, see the function [prototype](https://github.com/eProsima/micro-CDR/blob/50776f4ab7895e47af535a6d6e7ec697ddf443f3/src/c/microCdr.c#L1018)

Comment (bkueng) [reply msg/templates/uorb_microcdr/msg.cpp.template]: But then https://github.com/PX4/Firmware/pull/8888/files#diff-8190543ff07160bad612518e04f0e097R73 should be `uint64_t`

Comment (vicentemonge) [reply msg/templates/uorb_microcdr/msg.cpp.template]: No, there is no uint64_t type in IDL

Comment (bkueng) [reply msg/templates/uorb_microcdr/msg.cpp.template]: Ah, I see, that's IDL. You're right then.

**Comment by ChristophTobler** [comment]
@avinash-palleti and I have already tested this.

**Pull request merged**